.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: Unit Tests
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
What do you think of running the tests for every PR?

```suggestion

```

Reply:
I think it's a great ideia. Thanks!